examples/express-composition/src/server.ts

```diff
@@ -0,0 +1,49 @@
+import {NoteApplication} from './application';
+import {ApplicationConfig} from '@loopback/core';
+import {Request, Response} from 'express';
+import * as express from 'express';
+import * as path from 'path';
+import * as pEvent from 'p-event';
+import * as http from 'http';
+
+export class ExpressServer {
+  private app: express.Application;
+  public readonly lbApp: NoteApplication;
+  private server: http.Server;
+
+  constructor(options: ApplicationConfig = {}) {
+    this.app = express();
+    this.lbApp = new NoteApplication(options);
+
+    // Expose the front-end assets via Express, not as LB4 route
+    this.app.use('/api', this.lbApp.requestHandler);
+
+    // Custom Express routes
+    this.app.get('/', function(_req: Request, res: Response) {
+      res.sendFile(path.resolve('public/express.html'));
+    });
+    this.app.get('/hello', function(_req: Request, res: Response) {
+      res.send('Hello world!');
+    });
+
+    // Serve static files in the public folder
+    this.app.use(express.static('public'));
+  }
+
+  public async boot() {
+    await this.lbApp.boot();
+  }
+
+  public async start() {
+    await this.lbApp.start();
+    this.server = this.app.listen(3000);
+    await pEvent(this.server, 'listening');
+  }
+
+  // For testing purposes
+  public async stop() {
+    if (!this.server) return;
+    this.server.close();
```

Comment:
If we decide to start `lbApp` in `start()` above, then we should also stop it IMO.

Reply:
Will add this in a follow-up PR or just remove starting the LoopBack app depending on https://github.com/strongloop/loopback-next/pull/2306#discussion_r261250257